shared/constants/network.js

```diff
@@ -96,3 +96,6 @@ export const NATIVE_CURRENCY_TOKEN_IMAGE_MAP = {
   [TEST_ETH_SYMBOL]: TEST_ETH_TOKEN_IMAGE_URL,
   [BNB_SYMBOL]: BNB_TOKEN_IMAGE_URL,
 };
+
+export const INFURA_BLOCKED_MESSAGE =
+  'EthQuery - RPC Error - This service is not available in your country';
```

Comment:
Curious about this error message format - I don't think I've seen this type before. I guess you saw this in testing?

Reply:
As this is an EthQuery call, the error will be prefixed with `EthQuery - RPC Error - ` per: https://github.com/ethereumjs/eth-query/blob/master/index.js#L67

Then per https://consensys.slack.com/archives/GV90KPED9/p1618353684021400?thread_ts=1618348658.016900&cid=GV90KPED9, the full message will be `EthQuery - RPC Error - This service is not available in your country`

Reply:
Thanks, I was just able to trace it down. It all makes sense now. I was expecting the status code to be included with the error somewhere but apparently we don't do that yet, so I've opened an issue for that: https://github.com/MetaMask/eth-json-rpc-infura/issues/45

Reply:
++